app/controllers/ChatController.scala

```diff
@@ -57,6 +57,14 @@ class ChatController @Inject() (cc: ControllerComponents, chatService: ChatServi
     }
   }
 
+  def moveChatToTrash(chatId: String): Action[AnyContent] = authenticatedUserAction.async {
+    authenticatedRequest =>
+      chatService.moveChatToTrash(chatId, authenticatedRequest.userId).map {
+        case true => NoContent
```

Comment:
Don't pattern match on booleans. Use map(if (_) NoContent else NotFound)

Reply:
Is the use of ".map(if(_)...) " better just from a visual perspective, or is there a performance difference?